bundle.yaml

```diff
@@ -49,6 +49,7 @@ spec:
                         properties:
                           name:
                             description: Label to match.
+                            minLength: 1
```

Comment:
Sorry, not too familiar with how validations work in practice. What is minLenght here, number of chars or number of strings? Thanks!

Reply:
this the minimal string length, e.g. you can't have an empty label name.